src/navigation/Nav.jsx

```diff
@@ -164,7 +164,7 @@ export class Nav extends React.Component {
 		];
 
 		let authItems = [
-			self.isProMember && {
```

Comment:
Why are we replacing `isProMember` with `is_pro_admin`? Are we limiting the scope of something we're displaying? I am quite confused.

Reply:
naming convention due to old endpoint in chapstick

Reply:
`isProMember` is a parameter we defined because we were doing a separate API call to get the info. Now the info comes from `self` so we don't need to define our own separate variable.